vllm/metadata.csv

```diff
@@ -0,0 +1,44 @@
+vllm.num_requests.running,gauge,,,,Number of requests currently running on GPU.,0,vllm,,,
+vllm.num_requests.swapped,gauge,,,,Number of requests swapped to CPU.,0,vllm,,,
+vllm.num_requests.waiting,gauge,,,,Number of requests waiting.,0,vllm,,,
+vllm.process.cpu_seconds.count,count,,second,,Total user and system CPU time spent in seconds.,0,vllm,,,
+vllm.process.max_fds,gauge,,file,,Maximum number of open file descriptors.,0,vllm,,,
+vllm.process.open_fds,gauge,,file,,Number of open file descriptors.,0,vllm,,,
+vllm.process.resident_memory_bytes,gauge,,byte,,Resident memory size in bytes.,0,vllm,,,
+vllm.process.start_time_seconds,gauge,,second,,Start time of the process since unix epoch in seconds.,0,vllm,,,
+vllm.process.virtual_memory_bytes,gauge,,byte,,Virtual memory size in bytes.,0,vllm,,,
+vllm.prompt_tokens.count,count,,,,Number of prefill tokens processed.,0,vllm,,,
+vllm.python.gc.collections.count,count,,,,Number of times this generation was collected,0,vllm,,,
+vllm.python.gc.objects.collected.count,count,,,,Objects collected during gc,0,vllm,,,
+vllm.python.gc.objects.uncollectable.count,count,,,,Uncollectable objects found during GC,0,vllm,,,
+vllm.python.info,gauge,,,,Python platform information,0,vllm,,,
+vllm.request.generation_tokens.bucket,count,,,,Number of generation tokens processed.,0,vllm,,,
+vllm.request.generation_tokens.count,count,,,,Number of generation tokens processed.,0,vllm,,,
+vllm.request.generation_tokens.sum,count,,,,Number of generation tokens processed.,0,vllm,,,
+vllm.request.params.best_of.bucket,count,,,,Histogram of the best_of request parameter.,0,vllm,,,
+vllm.request.params.best_of.count,count,,,,Histogram of the best_of request parameter.,0,vllm,,,
+vllm.request.params.best_of.sum,count,,,,Histogram of the best_of request parameter.,0,vllm,,,
+vllm.request.params.n.bucket,count,,,,Histogram of the n request parameter.,0,vllm,,,
+vllm.request.params.n.count,count,,,,Histogram of the n request parameter.,0,vllm,,,
+vllm.request.params.n.sum,count,,,,Histogram of the n request parameter.,0,vllm,,,
+vllm.request.prompt_tokens.bucket,count,,,,Number of prefill tokens processed.,0,vllm,,,
+vllm.request.prompt_tokens.count,count,,,,Number of prefill tokens processed.,0,vllm,,,
+vllm.request.prompt_tokens.sum,count,,,,Number of prefill tokens processed.,0,vllm,,,
+vllm.request.success.count,count,,,,Count of successfully processed requests.,0,vllm,,,
+vllm.time_per_output_token.seconds.bucket,count,,second,,The duration of time per output token in seconds.,0,vllm,,,
+vllm.time_per_output_token.seconds.count,count,,second,,The count of time per output token in seconds.,0,vllm,,,
+vllm.time_per_output_token.seconds.sum,count,,second,,The sum of time per output token in seconds.,0,vllm,,,
+vllm.time_to_first_token.seconds.bucket,count,,second,,The duration of time to first token in seconds.,0,vllm,,,
+vllm.time_to_first_token.seconds.count,count,,second,,The count of time to first token in seconds.,0,vllm,,,
```

Comment:
In prometheus histograms, the buckets and counts aren't measure of seconds. They're the number of a observations that fit into the second buckets I.E 3 observations inside the 0.1 > x < 0.5 second le bucket for example.
```suggestion
vllm.time_to_first_token.seconds.bucket,count,,,,The duration of time to first token in seconds.,0,vllm,,,
vllm.time_to_first_token.seconds.count,count,,,,The count of time to first token in seconds.,0,vllm,,,
```